meetings/working-groups/collection-literals/Supporting-Interface-TargetTypes.md

```diff
@@ -0,0 +1,72 @@
+
+2. Our own best practices, which are aped in the ecosystem, are that (if contiguous data segments are not necessary) that apis be maximally permissive in what they take as arguments.  Put another way, our apis "accept the general, but return the specific".  For collections, the way that we "accept the general" is normally as `IEnumerable<T>` (which corresponds to the data above).
+
+3. There is a coherent, and explainable difference between "natural types" and interfaces.  While both relate to the idea of "a type must be picked for you", the needs and choices differ between the two.
+
+   For natural types we believe the primary case where it is picked is for `var`.  In the `var` case the code exists in a mutable location where users generally have full control to manipulate as they desire.  Collections can be used to buffer, manipulate, filter, and generally act as a temporary scratch location to produce a final result.  As such, a mutable type (like `List<T>`) is the most 'natural' type here given the need for general flexibility.  `List<T>` itself is the most commonly used collection type for this purpose, and it naturally aligns semantically with the sequence of elements that a collection expression syntactically represents.  Like `Dictionary<K,V>` it presents what the ecosystem thinks of as *the* collection type when trying to manipulate values.
+
+   Note that natural types do also appear both for dynamic and object.  However, It does not feel like these cases are substantively different or interesting enough to warrant a different 'natural type' (though the point is open for debate).
+
+   Conversely, we expect interface-target types to be most commonly used when passing data to other APIs, or when used to store data for properties the user has (and exposes) of those interface types.  In these cases, mutability is undesirable.  And, indeed, can be a source of problems (as external consumers might be able to change your data).
+
+   Because they core scenarios differ so wildly on their expectations around mutability and around the problems that arise (for either) by aligning on what the other does, it seems reasonable to state and explain things if we go down a path where there are different types and choices between natural and target types.
+
+4. Furthermore, alignment between the two systems (target and natural typing) could actually be highly negative if we promise a concrete collection type. For example, if we promise a `List<T>` type, then that eliminates the ability to use a singleton for the *very* common and *core* use-case of passing `[]` as an argument to `IEnumerable` apis (or initializing an `IEnumerable` property).  This would force users away from this, and require them to be as verbose as today to be explicit to get the desired savings.  This deeply undercuts are core narrative that you can use collection expressions and they can be the *efficient* type you can depend on.
```

Comment:
```suggestion
4. Furthermore, alignment between the two systems (target and natural typing) could actually be highly negative if we promise a concrete collection type. For example, if we promise a `List<T>` type, then that eliminates the ability to use a singleton for the *very* common and *core* use-case of passing `[]` as an argument to `IEnumerable<T>` apis (or initializing an `IEnumerable<T>` property).  This would force users away from this, and require them to be as verbose as today to be explicit to get the desired savings.  This deeply undercuts are core narrative that you can use collection expressions and they can be the *efficient* type you can depend on.
```